tests/test_solver.py

```diff
@@ -120,6 +120,41 @@ def _submit_and_check(self, solver, linear, quad, **kwargs):
 
         return results
 
+@unittest.skipUnless(config, "No live server configuration available.")
+class Utilities(_QueryTest):
+    """Test solver utility methods."""
+
+    def test_qpu_access_time_estimate(self):
+        with Client(**config) as client:
+            solver = client.get_solver(qpu=True)
+
+            runtime_q1000 = solver.estimate_qpu_access_time(num_qubits=1000)
+            # results for num_qubits=1000: Advanatge4.1 15341, DW_2000Q_6 11775
+            self.assertGreater(runtime_q1000, 5000)
+            self.assertGreater(50000, runtime_q1000)
```

Comment:
How about you verify the estimate by comparing it with actual access time (since this is a live test, you might as well sample)?

Reply:
That's tempting but the estimate varies by selected system, and available systems can change over time, and there can be significant variations (especially between chips with different topologies, but also the same topology), so that's why I set a large range.

Reply:
Added sampling versus estimate unit test